src/stylesheets/components/_page.scss

```diff
@@ -2,9 +2,23 @@ sds-page {
   @include u-display("block");
   @include u-margin-top(2);
   @include u-margin-bottom(8);
```

Comment:
We discussed page would have minheight and padding bottom only, should remove other styles.